### ISSUE TITLE
haskellPackages.hakyll does not configure against pandoc-2.7

### ISSUE THREAD

**Issue by endgame** [NixOS/nixpkgs#57479]
## Issue description

`hakyll.cabal` depends on `pandoc < 2.7`. Current nixpkgs-unstable contains `pandoc-2.7` as `haskellPackages.pandoc`. Current nixpkgs master has `pandoc-2.5` as `pandoc` and `pandoc-2.7` as `pandoc_2_7`.

Upstream bug for version bound relaxation: https://github.com/jaspervdj/hakyll/issues/693

**Comment by NickSeagull** [comment]
@endgame a workaround that I did was to override the pandoc of the package set with the `2.6` one:

```nix
pkgs.haskell.packages.${compiler}.extend(self: super: {
  pandoc = pkgs.haskell.lib.dontCheck (self.callHackage "pandoc" "2.6" {});
})
```

**Comment by endgame** [comment]
Cheers. I'm currently patching hakyll's cabal file as my workaround.

**Comment by erictapen** [comment]
This is adressed in #57587.

**Comment by erictapen** [comment]
This can be closed now due to #57587.
